src/legacy/core_plugins/timelion/public/timelion_vis_fn.ts

```diff
@@ -22,6 +22,7 @@ import { i18n } from '@kbn/i18n';
 import { ExpressionFunction, KibanaContext, Render } from 'src/plugins/expressions/public';
 import { getTimelionRequestHandler } from './vis/timelion_request_handler';
 import { TimelionVisualizationDependencies } from './plugin';
+import { TIMELION_VIS_NAME } from './vis';
```

Comment:
nit: link to vis/index not looks good to me. Maybe we can pass it from `TimelionVisualizationDependencies`

Reply:
its part of the same plugin (internal code), no need to pass it in as dependency